src/librustc/ty/layout.rs

```diff
@@ -246,13 +246,22 @@ impl<'tcx> LayoutCx<'tcx, TyCtxt<'tcx>> {
         let align = a.value.align(dl).max(b_align).max(dl.aggregate_align);
         let b_offset = a.value.size(dl).align_to(b_align.abi);
         let size = (b_offset + b.value.size(dl)).align_to(align.abi);
+
+        // HACK(nox): We iter on `b` and then `a` because `max_by_key`
```

Comment:
Why @nox here? According to git all comments are authored by @eddyb.

Reply:
If you search for "nox" in this diff you can see where the code was copied from

Reply:
Missed that :)